src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKStyle.java

```diff
@@ -888,10 +888,7 @@ else if ("CheckBox.iconTextGap".equals(key) ||
         }
 
         // Is it a stock icon ?
-        GTKStockIcon stockIcon = null;
-        synchronized (ICONS_MAP) {
-            stockIcon = ICONS_MAP.get(key);
-        }
+        GTKStockIcon stockIcon = ICONS_MAP.get(key);
 
         if (stockIcon != null) {
```

Comment:
```suggestion
        GTKStockIcon stockIcon = ICONS_MAP.get(key);
        if (stockIcon != null) {
```
Maybe remove the blank line? It used to separate a block where `stockIcon` was initialised and the condition. Now it's better to keep these two lines closer.